Sources/MapboxNavigation/PuckType.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import MapboxMaps
+
+public enum PuckType {
```

Comment:
Historically, we’ve only used the term “puck” informally. In most other contexts, developers encounter terms such as “user location indicator” or “user course view” that are independent of the specific look and feel. Consider renaming this type to `UserLocationStyle` to allow for a look and feel that looks nothing like the traditional hockey puck. Even though the map SDK is using the “puck” terminology more formally these days, a UserCourseView could look like anything, and the term “type” is even more overloaded than “style”.

Reply:
Thanks for the suggestion! I'll change the name to the `UserLocationStyle `.